packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/OutputTypes/Parameter.cs

```diff
@@ -60,7 +60,7 @@ public Parameter(InputParameter inputParameter)
             DefaultValue = defaultValue;
         }
 
-        internal static readonly IEqualityComparer<Parameter> EqualityComparerByType = new ParameterByTypeEqualityComparer();
+        public static readonly IEqualityComparer<Parameter> EqualityComparerByType = new ParameterByTypeEqualityComparer();
```

Comment:
Instead of this is there any reason we don't just have Parameter implement `IEquatable<Parameter>`?